letsencrypt-win-simple/Plugin/ManualPlugin.cs

```diff
@@ -107,7 +107,14 @@ public override void HandleMenuResponse(string response, List<Target> targets)
                 if (Program.Options.San)
                 {
                     Console.Write("Enter all Alternative Names seperated by a comma ");
-                    Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
+
+                    //Console.SetIn(new System.IO.StreamReader(Console.OpenStandardInput(8192)));
```

Comment:
You don't need to leave the old code in there since you are replacing it.

Otherwise it looks and works great. It accepted up to 16382 characters for me. That's up to 162 characters per domain up to the 100 domain limit. If someone goes over that, I think they need shorter domains.